### PR TITLE
feat(Header): Add link to header logo

### PR DESCRIPTION
**Pull request by ZhenYongSeekAsia**
** Please provide as much detail as possible, but feel free to remove irrelevant sections **

Add link to header's logo
image

BREAKING CHANGE:

{{ Description of breaking API change }}

RFC URL:

{{ URL of associated RFC for API change }}

MIGRATION GUIDE:

Before:

```js
{{ Old code }}
```

After:

```js
{{ New code }}
```

EXAMPLE USAGE:

```js
{{ Code }}
```

### CODE REVIEW

File: react/Header/Header.less [hunk 1]
```diff
@@ -72,3 +72,7 @@
 .primaryNavLink {
   text-decoration: none;
 }
+
+.logo {
+  display: inline-block;
```
Comment (mengtzu) [inline react/Header/Header.less]: Why is this necessary?

File: react/Header/__snapshots__/Header.test.js.snap [hunk 2]
```diff
@@ -42,9 +42,12 @@ exports[`Header should pass through optional props 1`] = `
   <div
     className="primaryNavNoLogin"
   >
-    <FakeComponent
-      country="my"
-    />
+    <a>
```
Comment (mengtzu) [inline react/Header/__snapshots__/Header.test.js.snap]: No href provided to the test?

**Comment by mengtzu** [comment]
Nice simple little PR <3

Have we confirmed with design that this is the behaviour they want?  Seems pretty obvious but just want to make sure.

**Comment by ZhenYongSeekAsia** [comment]
@mengtzu i think the design no changes, but make the Logo into a link.

**Comment by ZhenYongSeekAsia** [comment]
http://seek-asia-style-guide--b4427e73a7a3acf03871336b75284d2869b47b4a.surge.sh/jobsdb-header

**Comment by ZhenYongSeekAsia** [comment]
[Update brand logo to link to SRP homepage (empty state)](https://trello.com/c/Z2BCrm3x/688-update-brand-logo-to-link-to-srp-homepage-empty-state)
